corehq/apps/domain/views/releases.py

```diff
@@ -181,3 +185,30 @@ def update_release_restriction(request, domain, restriction_id, active):
                                if release.deactivated_on else None),
         }
     return HttpResponse(json.dumps(response_content), content_type='application/json')
+
+
+def toggle_release_restriction_by_app_profile(request, domain, restriction_id):
```

Comment:
This view also needs a permissions decorator

Reply:
https://github.com/dimagi/commcare-hq/pull/24692/commits/d301beed97497fcd7780f1003a53a8456240eacd